packages/react-native/Libraries/AppDelegate/RCTAppDelegate.mm

```diff
@@ -141,17 +155,18 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
   return nil;
 }
 
-- (NSDictionary *)prepareInitialProps
+- (void)updateInitialProps:(NSMutableDictionary *)initialProps
 {
-  NSMutableDictionary *initProps = self.initialProps ? [self.initialProps mutableCopy] : [NSMutableDictionary new];
-
 #ifdef RCT_NEW_ARCH_ENABLED
   // Hardcoding the Concurrent Root as it it not recommended to
   // have the concurrentRoot turned off when Fabric is enabled.
-  initProps[kRNConcurrentRoot] = @([self fabricEnabled]);
+  initialProps[kRNConcurrentRoot] = @([self fabricEnabled]);
 #endif
+}
```

Comment:
Ah, this method is now completely unused, can you remove it?

Reply:
Yeah, I also found it, removed already :)